src/libraries/System.Security.Cryptography.Encoding/tests/Cbor/CborReader.SkipValue.cs

```diff
@@ -9,22 +9,58 @@ namespace System.Formats.Cbor
 {
     public partial class CborReader
     {
-        public void SkipValue(bool validateConformance = false) => SkipToAncestor(0, validateConformance);
-        public void SkipToParent(bool validateConformance = false)
+        /// <summary>
+        ///   Reads the contents of the next value, discarding the result and advancing the reader.
+        /// </summary>
+        /// <param name="disableConformanceLevelChecks">
+        ///   Disable conformance level validation for the skipped value,
+        ///   equivalent to using <see cref="CborConformanceLevel.Lax"/>.
```

Comment:
Boolean parameters get a special construction.  https://github.com/dotnet/dotnet-api-docs/wiki/Parameters%3A-Methods-and-constructors

```suggestion
        ///   <see langword="true"/> to disable conformance level validation for the skipped value,
        ///   equivalent to using <see cref="CborConformanceLevel.Lax"/>; otherwise, <see langword="false"/>.
```

(Or you can expand the false case if it feels more complex than "otherwise":

```suggestion
        ///   <see langword="true"/> to disable conformance level validation for the skipped value,
        ///   equivalent to using <see cref="CborConformanceLevel.Lax"/>;
        ///   <see langword="false"/> to use the current conformance level to validate the value. 
```